lib/fortnox/api/models/order.rb

```diff
@@ -19,6 +19,13 @@ class Order < Fortnox::API::Model::Base
 
         # OrderRows Separate object
         attribute :order_rows, Array[OrderRow]
+
+        # Override to_hash to convert nested OrderRows to hash correctly.
+        def to_hash *args
```

Comment:
I did. But feel free to give it another try :) Just remove that method and see if the tests passes.